stytch/b2b/sessions.go

```diff
@@ -351,15 +381,33 @@ func marshalJWTIntoSession(claims sessions.Claims) (*sessions.MemberSession, err
 	}
 	expires = expires.UTC()
 
-	return &sessions.MemberSession{
-		MemberSessionID:       claims.Session.ID,
-		MemberID:              claims.RegisteredClaims.Subject,
-		StartedAt:             &started,
-		LastAccessedAt:        &accessed,
-		ExpiresAt:             &expires,
-		AuthenticationFactors: claims.Session.AuthenticationFactors,
-		OrganizationID:        claims.Organization.ID,
+	return &sessions.LocalJWTResponse{
+		MemberSession: &sessions.MemberSession{
+			MemberSessionID:       claims.Session.ID,
+			MemberID:              claims.RegisteredClaims.Subject,
+			StartedAt:             &started,
+			LastAccessedAt:        &accessed,
+			ExpiresAt:             &expires,
+			AuthenticationFactors: claims.Session.AuthenticationFactors,
+			OrganizationID:        claims.Organization.ID,
+		},
+		RolesClaim: claims.Roles,
 	}, nil
 }
```

Comment:
This feels wrong -- I think we want to still return a MemberSession but then later check the roles claim from the `claims` object.